lib/matplotlib/backends/backend_wxagg.py

```diff
@@ -71,15 +71,8 @@ def blit(self, bbox=None):
     filetypes = FigureCanvasAgg.filetypes
 
 
-@cbook.deprecated("2.2")
```

Comment:
Please leave until 3.0 (one minor-release deprecation period at least, usually two).

Reply:
I don't see that any other NavigationToolbar2 is deprecated, especially not the wx non-Agg one.
With the derived class removed and the modified `import NavigationToolbar2Wx as NavigationToolbar2WxAgg` things are consistent again (no deprecation).

Not having the Toolbar alias in backend_wxagg can break things, though.

Reply:
Having a deprecation for the Toolbar alias would require to replace the assignment `Toolbar = NavigationToolbar2Wx` with a derived and deprecated class. I would now be in favor of having this in wx and wxagg. What do you think?

Reply:
derived deprecated class sounds good to me.

Reply:
Thanks. Done, incl. alternatives in deprecation messages.